src/stages.md

```diff
@@ -6,7 +6,7 @@ The implementation sequence is going to be the following:
     1. Add support of Conan package manager to [scale-code-cpp](https://github.com/soramitsu/scale-codec-cpp)
     2. Add support of Conan package manager to [cpp-libp2p](https://github.com/libp2p/cpp-libp2p)
     3. Create light client application with transaction management functionality.
-    4. Add support of *GetStorage* and *GetStorageProof* JSON RPC methods
+    4. Add support of `chainHead_unstable_follow`, `chainHead_unstable_unfollow` and `chainHead_unstable_unpin` JSON RPC methods
```

Comment:
why is this different from the list above?

Reply:
Because in old API you don't need to call some "follow" methods to get storage value, just connect to a chain and get value by block hash and key. In this API without calling `chainHead_unstable_follow` you aren't able to do anything. So it shoud be implemented first